### PR TITLE
census 0.8.22 ❄️ 

### PR DESCRIPTION
**Pull request by psteyer**
census 0.8.22 :snowflake:

**Destination channel:** Snowflake

### Links

- [PKG-5104](https://anaconda.atlassian.net/browse/PKG-5104) 
- [Upstream repository](https://github.com/datamade/census/tree/v0.8.22)

### Explanation of changes:

- Wasn't able to get tests to work due to them testing API functionality and would run out of calls

[PKG-5104]: https://anaconda.atlassian.net/browse/PKG-5104?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,21 +10,20 @@ source:
   sha256: ed7d092c72fad9f6f839e1501229490cb278403ae95529a02860f3012f02bc0a
 
 build:
-  noarch: python
   number: 0
```
Comment (lorepirri) [inline recipe/meta.yaml:13]: @psteyer conda-forge has the same version released for this package, could you please increase the build number for ours?

```suggestion
  number: 1
```

**Review by lorepirri — COMMENTED** [review]
`abs.yaml` missing for SF channel

File: recipe/meta.yaml (hunk 1 above)
Comment (lorepirri) [reply recipe/meta.yaml:13]: or maybe I remember wrong, @skupr-anaconda ?

Comment (skupr-anaconda) [reply recipe/meta.yaml:13]: The channel has higher priority. So there is no correlation between the package build number and the channel. If the package is available in the ❄️  channel then the solver chooses it and ignores the conda-forge. 
In short, for a new package, the build number should be set to 0 as our common practice.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -36,12 +35,16 @@ test:
     - pip
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:35]: Could you run pytest as the `tests` folder is present in the census module? 
`- pytest -v --pyargs census  `

Comment (lorepirri) [reply recipe/meta.yaml:35]: Most of the tests call an endpoint and after a few calls the server rejects the calls with "too many requests". 

Either we call 1 test of that kind only (per platform, per python version).

Despite all of this, a smoke test is welcome :)

File: recipe/meta.yaml (hunk 1 above)
Comment (lorepirri) [reply recipe/meta.yaml:13]: Got it, thank you. I'll ask more info on slack.

Comment (psteyer) [reply recipe/meta.yaml:13]: Just to double check, we will be keeping it at `number: 0` then, since it's the first build for the channel?

Comment (skupr-anaconda) [reply recipe/meta.yaml:13]: yes

**Review by skupr-anaconda — APPROVED** [review]

**Review by lorepirri — APPROVED** [review]

**Pull request merged**